public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -3,7 +3,6 @@ import { LogRows, CustomScrollbar } from '@grafana/ui';
 import { LogsDedupStrategy, PanelProps } from '@grafana/data';
```

Comment:
Change of SortOrder import as we have moved it to grafana/data. Plus we are just passing order of logs and the ordering is processed in LogRows - the same way as in explore.